.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 22.9.0
```

Comment:
Do you know which tools us this file?

Reply:
Originally it was comes from `asdf` but the great thing is that the official https://github.com/actions/setup-node does support it too, making so easy to control the version of node being used locally and on CI too. Noted It does not override a manually configured version in the action.

Personally, I would configure cspell-dicts repo in particular to use a single version of node. That repo does not really need testing with multiple versions as it builds only dictionaries. For the main cspell repo, that is another story.

Controlling the version should lower the chance of getting surprises, aka more stable CI/CD pipelines.

Reply:
@ssbarnea,

Cool! I didn't know that.